pkgs/development/python-modules/virtual-glob/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  flit-core,
+  lib,
+  pytest,
+}:
+
+buildPythonPackage rec {
+  pname = "virtual-glob";
+  version = "0.2.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "virtual_glob";
+    inherit version;
+    hash = "sha256-FoHK4giC3pWRxmO4LRVgGduWy1PBgMZkhTyhWpcHvdI=";
+  };
```

Comment:
Should probably switch to the [GitHub source](https://github.com/chrisjsewell/virtual-glob) as it contains tests (tests are commonly stripped from PyPI source releases)